package-managers/qubes-download-dom0-updates.sh

```diff
@@ -129,10 +145,11 @@ if [ ${#PKGLIST[@]} -eq 0 ] && [ "$CHECK_ONLY" = "1" ]; then
 fi
 
 # now, we will download something
-YUM_COMMAND=(fakeroot "${YUM[@]}" "$YUM_ACTION" --downloadonly)
+UPDATE_COMMAND=(fakeroot $UPDATE_CMD $UPDATE_ACTION "${UPDATE_ARGUMENTS[@]}")
+"$UPDATE_CMD" "$UPDATE_ACTION" --help | grep -q downloadonly && UPDATE_COMMAND+=(--downloadonly)
```

Comment:
What if the option is not there? In absence of an alternative block for this, it will try to actually install packages, which will fail. But since yumdownloader fallback is not relevant anymore, I think you can keep adding --downloadonly unconditionally. And if it's missing for some reason, let it fail with a message about it.

Reply:
> I think you can keep adding --downloadonly unconditionally

This will break `qubes-dom0-update --action=search` and `qubes-dom0-update --action=list.`. List and search actions will fail with `--downloadonly` option.

Reply:
Right, so this is a bit smarter detection which action supports the option. Can you add a comment about that (including info that it used to be a global option in DNF4)?

Reply:
Sure. I  will do it